leshan-server-redis/src/main/java/org/eclipse/leshan/server/redis/RedisSecurityStore.java

```diff
@@ -215,7 +221,7 @@ public void setEndpointByPskIdKey(String endpointByPskIdKey) {
          * Set the prefix for all keys and prefixes including {@link #securityInfoByEndpointPrefix} and
          * {@link #endpointByPskIdKey}.
          * <p>
-         * Default value is {@literal SECSTORE#}.
+         * Default value is {@literal SECSTORE#}. Should not be {@code null}, can be empty.
```

Comment:
Here it is said : `Should not be {@code null}`

But looking at : 
```java
    if (this.prefix != null) {
                this.securityInfoByEndpointPrefix = this.prefix + this.securityInfoByEndpointPrefix;
                this.endpointByPskIdKey = this.prefix + this.endpointByPskIdKey;
            }
```
You manage `null`.

I think it's OK to support `null` prefix.

Reply:
Forgot about it, wrote a JavaDoc first, then added a null-check.

Reply:
No problem, this happens. An easy way to limit this kind of issue is to do a review of you code, like if you review someone else's code. :wink:

Reply:
This is actually one of the "rules" in my team at work. Don't know, why I didn't follow it while preparing this PR.
Will pay more attention next time.